internal/database/dialer/cockroachdb/table/staker_profit_snapshot.go

```diff
@@ -9,14 +9,14 @@ import (
 )
 
 type StakerProfitSnapshot struct {
-	ID               uint64          `gorm:"column:id"`
-	Date             time.Time       `gorm:"column:date"`
-	EpochID          uint64          `gorm:"column:epoch_id"`
-	OwnerAddress     common.Address  `gorm:"column:owner_address"`
-	TotalChipAmounts decimal.Decimal `gorm:"column:total_chip_amounts"`
-	TotalChipValues  decimal.Decimal `gorm:"column:total_chip_values"`
-	CreatedAt        time.Time       `gorm:"column:created_at"`
-	UpdatedAt        time.Time       `gorm:"column:updated_at"`
+	ID              uint64          `gorm:"column:id"`
+	Date            time.Time       `gorm:"column:date"`
+	EpochID         uint64          `gorm:"column:epoch_id"`
+	OwnerAddress    common.Address  `gorm:"column:owner_address"`
+	TotalChipAmount decimal.Decimal `gorm:"column:total_chip_amount"`
+	TotalChipValue  decimal.Decimal `gorm:"column:total_chip_value"`
```

Comment:
manually rename the column fields or use a goose migration script to make the changes.